Domain Connect Spec Draft.adoc

```diff
@@ -1583,29 +1586,34 @@ display to them the specific records that are being set in DNS.
 * If there are conflicts, either at the template or record level, the
 consent UX should warn the user about these conflicts. For templates,
 this would be services that would be disabled. For records, this would be
-records that would be overwritten. This could be progressively disclosed.
-
-Note: When re-applying the same template, ideally DNS Providers would not detect
-the conflict. Instead, the first template would be removed and the new
-instance applied. For most templates this is a benign operation, unless
-the template contains variables in host names. For consideration
-of this, see the section below.
+records that would be deleted or overwritten. This could be progressively disclosed.
 
 === Record Types and Conflicts
 
-The proposed handling of records and conflicts is as follows (if not
-otherwise specified, conflicts occur if the records have the same name):
+Conflict detection done by DNS provider prior to template application has to take
+into consideration specifics of each DNS record type. The rules outlined below shall
+assure predictable conflict resolution between DNS providers. Each rule applies to
+the records on the very same host, unless specifed otherwise.
+
+* CNAME records conflicts with all other records
+* any record type conflicts with existing CNAME record
+* MX, APEXCNAME, SRV records always conflict with records of the same type
+* A and AAAA records conflict with any other A and/or AAAA record, to avoid IPv4
+and IPv6 pointing to different services.
+* TXT records conflict detection is handled looking at txtConflictMatchingMode
+parameter
+** None: This indicates that the TXT records do not conflict with any other TXT
+record. This is the default setting, if not specified.
+** All: This indicates that the TXT records conflict with any other TXT record
+** Prefix: This indicates that TXT record conflict with any other TXT containing value starting with txtConfictMatchingPrefix 
+
+=== Re-apply
+
+When applying the same template again, identified by the same providerId, serviceId,
+domain and host, ideally DNS Providers would not detect the conflict. Instead, the records
+created by the initial template application would be removed and the new instance applied.

```

Comment:
I might add something that indicates this is optional and up to the DNS Provider.

Reply:
added additional language in 6d45f5dfe78776e4b0c2d279dffe982e383a36b3